### ISSUE TITLE
Discussion: getting started

### ISSUE THREAD

**Issue by blahah** [BioTest/core#1]
BioTest sounds like a great idea. I just wanted to open up some discussion about how to get going.

In the [BioJulia](https://github.com/BioJulia/Bio.jl) project, which is currently in pre-release infancy, we are planning to implement a peer-reviewed package system. The idea is that BioJulia will include a package distribution system to complement that of the Julia language, similar to what BioConductor does. The difference will be that when a package is submitted to BioJulia, the code will be subjected to rigorous peer review, and held to high standards (correctness, novelty, test coverage, documentation).

It seems to me that BioTest and the system we are planning could interoperate very nicely. Our standards could be compatible, and our reviewer community could be officially part of BioTest, and a subset of it.

Apart from actually implementing our package system, the key challenges for us to launch in BioJulia are:
- attract and organise a community of reviewers
- define the processes and standards by which reviews are conducted (some of which will be Julia specific)

For BioTest, those challenges exist at a higher level - there needs to be a potentially much larger community with expertise in a variety of languages and technologies, as well as some review processes and standards that are language agnostic.

I propose that the way forward could start with:
1. Creating an RFC (e.g. BioTest/rfc), so that interested parties can start developing things
2. Reaching out to all the bio\* projects to start raising awareness and recruiting community members

**Comment by blahah** [comment]
Bump - anyone out there?

**Comment by btemperton** [comment]
Yep, sorry for the delay. Am currently drowning in grant applications. Will be resurfacing in 2 weeks. I think the ideas you propose are great and certainly want to explore them further.

**Comment by btemperton** [comment]
Didn't mean to close this!

**Comment by blahah** [comment]
OK great - good luck with the grants!

**Comment by MDSharma** [comment]
bump.. let's do this.
